.github/dependabot.yml

```diff
@@ -1,71 +0,0 @@
-version: 2
-updates:
```

Comment:
why did you remove all these?

Reply:
Dependabot will still give us security updates- this prevents the nagging PRs